docs/GroupDefs.hpp

```diff
@@ -59,8 +59,8 @@
  */
 
 /*!
- * \defgroup EmbeddingMaps Embedding Maps
- * A collection of embedding maps used to construct computational domains.
+ * \defgroup CoordinateMaps Coordinate Maps
```

Comment:
Please delete these two lines; there is already a group above for coordinate maps